executor_manager/tests/executors/test_docker_executor.py

```
@@ -323,3 +323,236 @@ def test_call_callback_error(self, executor):
+        )
+
+        with pytest.raises(RuntimeError, match="Container health check timed out"):
+            executor._check_container_health(
+                task, executor_name, is_validation_task=False
+            )
+
+        # Verify cleanup was attempted
+        # There should be at least one call trying to clean up
+        cleanup_calls = [
+            call
+            for call in mock_subprocess.run.call_args_list
```

Ruff (0.14.11)

Redefinition of unused `call` from line 6: `call` redefined here

(F811)